ui/src/widgets/ui-form/UIForm.vue

```diff
@@ -66,6 +70,30 @@ export default {
     },
     methods: {
         onSubmit: function () {
+            // Prevent sending null for switch and combobox, if type number send as Number or null if nothing present on text field and if other fields not present, send empty string
+            const option = this.options
+            option.forEach(opt => {
+                if (opt.type === 'checkbox' || opt.type === 'switch') {
+                    if (this.input[opt.key] === undefined || this.input[opt.key] === null) {
```

Comment:
```suggestion
                    if (typeof(this.input[opt.key]) === 'undefined' || this.input[opt.key] === null) {
```

I get told off by @Steve-Mcl every time I do this, so just to make sure we're consistent across the app, checking `type` is a stronger check than checking the value.